services/index.js

```diff
@@ -0,0 +1,40 @@
+function calculateAge({ dob }) {
+  console.time("calculateAge")
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/gthankgod/howold/pullRequest?prid=9951845)
```suggestion
  console.time("calculateAge");
```